README.md

```diff
@@ -25,8 +25,8 @@ var mediator = require('fh-wfm-mediator');
 var Topics = require('fh-wfm-mediator/lib/topics');
 
 var topics = new Topics(mediator)
-  .prefix('wfm')
-  .entity('user')
+  .withPrefix('wfm')
+  .withEntity('user')
```

Comment:
Super super minor, but I noticed there are some inconsistencies with semi-colon usage in these code examples. This line and line 75, for instance, don't have semis, but line 25 does.